<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infoPageComp</class>
 <widget class="QDialog" name="infoPageComp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>421</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Calibri Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="infoCompName"/>
      </item>
      <item>
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Calibri Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Type:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="infoCompType"/>
      </item>
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Calibri Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Build Year:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="infoCompBuiltYear"/>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="font">
         <font>
          <family>Calibri Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nationality:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="infoCompNationality"/>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>Calibri Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Info:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="infoCompInfo"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2" rowstretch="0" columnminimumwidth="0">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="infoCompPic">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>200</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Mynd&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
